static/monorail.yml

```diff
@@ -1432,7 +1437,7 @@ paths:
         - name: q
           in: query
           description: query object to pass through to waterline.
-          required: false
+          required: true
```

Comment:
The on-http 1.1 API implementation doesn't use swagger in any capacity.  The source of that validation error message is not RackHD.

When the python library is updated to use this change (occurs post commit), this should break our functional tests here: https://github.com/RackHD/RackHD/blob/master/test/tests/api/v1_1/lookups_tests.py#L61.